Code/GraphMol/FileParsers/test1.cpp

```diff
@@ -5361,8 +5363,8 @@ int main(int argc, char *argv[]) {
 
   BOOST_LOG(rdInfoLog) << " ---- Running with German locale ----- "
                        << std::endl;
-  setlocale(LC_ALL, "de_DE.UTF-8");
-  std::cout << setlocale(LC_ALL, nullptr) << std::endl;
+  std::setlocale(LC_ALL, "de_DE.UTF-8");
+  std::cout << std::setlocale(LC_ALL, nullptr) << std::endl;
   testLocaleSwitcher();  // must be the last test
```

Comment:
I'm fairly sure that the // must be the last test comment is no longer necessary.